### PR TITLE
doc: Adding description about RADOS and RBD

### PR DESCRIPTION
**Pull request by dsdeeptisharma**

### CODE REVIEW

File: doc/mon/README.txt [hunk 1]
```diff
@@ -1,10 +1,12 @@
-paxos-call-chain.dot describes to some detail the call chain involved in the
+paxos-call-chain.dot describe some details about the call chain being involved in the
```
Comment (amitkumar50) [inline doc/mon/README.txt]: Should be "describes"

File: doc/glossary.rst [hunk 2]
```diff
@@ -7,6 +7,14 @@ Ceph is growing rapidly. As firms deploy Ceph, the technical terms such as
 that explain what each component does. The terms in this glossary are 
 intended to complement the existing technical terminology.
 
+- Defining some of these technical terms:
+"RADOS" : It stands for Reliable Autonomic Distributed Object Store (RADOS) which is an open source object storage service that is an important part of the Ceph distributed storage system. RADOS was created at the University of California at Santa Cruz. The Ceph open source community develops and maintains the software.
```
Comment (amitkumar50) [inline doc/glossary.rst]: @dsdeeptisharma Sorry to say, this does not speaks shortly about what RADOS actually is/What it does?
Only tells its full form/creators.
Your Thoughts?

Though RBD definition is bit more clear about what it does..

File: doc/glossary.rst [hunk 3]
```diff
@@ -7,6 +7,14 @@ Ceph is growing rapidly. As firms deploy Ceph, the technical terms such as
...
+
+"RBD" : It stands for Ceph’s RADOS Block Devices (RBD) interact with OSDs using kernel modules. An on-screen display (OSD) is a control panel on a computer monitor or television screen that allows you to select viewing options and/or adjust components of the display, such as brightness, contrast, and horizontal and vertical positioning.
+You can refer to this link: http://docs.ceph.com/docs/master/rbd/rbd/
```
Comment (amitkumar50) [inline doc/glossary.rst]: @dsdeeptisharma 
http://docs.ceph.com/docs/master/rbd/rbd/
404 Not Found

File: admin/Readme.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
+#Pybind of rgw 
+PyBind is a tool which can be used to generate python bindings for C or C++ APIs. pybind11 is a lightweight header-only library
+that exposes C++ types in Python and vice versa, mainly to create Python bindings of existing C++ code.
```
Comment (amitkumar50) [inline admin/Readme.md]: @dsdeeptisharma Is there need to place line beginning with "that  exposes C++ types in Python" in new line? I believe it was continuous sentence?

File: admin/Readme.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+Its goals and syntax are similar to the excellent [Boost python](http://www.boost.org/doc/libs/1_58_0/libs/python/doc/).
```
Comment (amitkumar50) [inline admin/Readme.md]: @dsdeeptisharma Sorry to say, what is excellent in boost python?

**Comment by gregsfortytwo** [comment]
Closing this PR as commits don't meet our formatting requirements and it has outstanding review comments.

**Pull request closed without merging**